e2e-test/tsps_e2e_test.py

```diff
@@ -219,9 +257,20 @@ def download_with_azcopy(sas_url):
                                     "https://github.com/DataBiosphere/terra-scientific-pipelines-service/blob/main/pipelines/testing/ImputationBeagleEmpty.wdl",
                                           azure_user_token)
 
-    # launch tsps imputation pipeline
-    logging.info("running imputation pipeline")
-    job_id = run_imputation_pipeline(tsps_url, azure_user_token)
+    # prepare tsps imputation pipeline run
+    logging.info("preparing imputation pipeline run")
+    job_id, pipeline_file_inputs = prepare_imputation_pipeline(tsps_url, azure_user_token)
+
+    # make sure we got a writable sas url
+    for key, value in pipeline_file_inputs.items():
+        logging.info(f"attempting to upload a file to {key} input")
+        write_sas_url = value['sasUrl']
+        upload_file_with_azcopy(value)
+        logging.info("successfully uploaded file")
+
+    # start pipeline run
+    logging.info("starting imputation pipeline run")
+    job_id = start_imputation_pipeline(job_id, tsps_url, azure_user_token)
```

Comment:
should we assert this job_id is the same as the job_id passed in and the job_id returned by `prepare_imputation_pipeline` ?